### PR TITLE
Peer to peer review process pull request #2

### PR DESCRIPTION
**Pull request by efmomoh**
Kindly review my code commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Code-review-repo
+PEER TO PEER REVIEW
 In this repo, I will be reviewing my own code. However, your contributions are highly welcomed!
```
Comment (TiteLunda) [inline README.md:3]: Kindly make a professional read me file. It helps your PR to look professional 😎

**Comment by efmomoh** [comment]
Kindly add descriptive text on the far-right of the listed videos

**Review by TiteLunda — COMMENTED** [review]
STATUS: REQUIRED CHANGES ♻️♻️

Hi [@efmomoh ](https://github.com/efmomoh)
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
To highlight 🍾

    All linters are passing ✔️
    Good commit history ✔️
    Follow GitHub flow ✔️

Required changes ♻️ ♻️

    Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@Tite](https://github.com/Titelunda) when you comment, so I can receive the notification or use Slack with the same name.

File: README.md (hunk 1 above)
Comment (efmomoh) [reply README.md:3]: ok, thanks for the feedback.

**Pull request closed without merging**